.github/workflows/ubuntu.yml

```diff
@@ -26,6 +26,9 @@ jobs:
   pytest-coverage: # from pytest_coverage.yml
     needs: lint
     runs-on: ubuntu-latest
+    env:
+      # A workaround for long FQDN names provided by GitHub actions.
+      FQDN: "localhost"
```

Comment:
Same here, let's make it a global setting for `ubuntu.yml`.

Reply:
Done.